### DISCUSSION TITLE
Collapsable/group output by command

### DISCUSSION THREAD

**Discussion by lafriks** [woodpecker-ci/woodpecker#2263] [Request a Feature]
### Clear and concise description of the problem

Currently executed command with trace script is being added `+ ` prefix and in large output it's hard to see what output is for what command if multiple commands is used for one step

### Suggested solution

Replace `+ ` with something more unique, example `❯ `? And display in log similarly how github actions does display steps.

### Validations

- [X] Checked that the feature isn't part of the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by lafriks** [comment]
Additionally we could recognize some common utilities and prefix with svg icons for nicer view (docker, go, node, shell etc)

**Comment by anbraten** [comment]
That would be really cool 🤩

**Comment by 6543** [comment]
as we do send logentrys via list ... and have different type
we could send a logentry with a new type ... to indicate there is a new command
